curbs/examples.md

```diff
@@ -9,11 +9,11 @@ This file presents a series of CDS [Curbs](/curbs) endpoint examples to use as t
 
 ## Curb Zones Minimum
 
-A [Query Curb Zones](/curbs#query-curb-zones) example of `/curbs/zone` with minimum required fields returned, plus a max parking stay of 60 minutes, and some parameters passed in.
+A [Query Curb Zones](/curbs#query-curb-zones) example of `/curbs/zone` with minimum required fields returned, and some parameters passed in.
```

Comment:
what are the possible parameters? might be nice to provide some examples of them here

Reply:
oh sorry I see that they're defined in the README, but would be helpful to link `some parameters` to https://github.com/openmobilityfoundation/curb-data-specification/tree/a1eb0ee503fb7f49adfa40223038db4ae8f392e5/curbs#query-parameters

Reply:
We've got the two parameters ?include_geometry=true&timestamp=1552678594427 in the example. Are you thinking some of the other ones too, like the latitude and longitude ones? We do plan to add more examples before the final version so there is a chance to do that elsewhere.

Reply:
yeah, more examples would be good, but also just a direct link to the full list attached to `some parameters` for reference -- it wasn't immediately obvious to me what "some parameters" was referring to

Reply:
Ah, good call, thanks! Will do.

Reply:
Done